bevaring_cli/commands/session.py

```diff
@@ -35,15 +35,15 @@ class SessionCmd(Cmd):
     def __attrs_post_init__(self):
         super().__init__()
         self.register(self.checkout, self.aws)
-        self._main.add(self._app, name='session', help="Manages process of new generation creation.")
+        self._main.add(self._app, name='session', help="Manages process of new generation creation"),
 
     def checkout(
         self,
-        datasett_id: str = Argument(..., help="Identifier of the dataset to check out."),
-        email: str = Argument(..., help="Email address where to send progress notification."),
+        datasett_id: str = Argument(..., help="Identifier of the dataset to check out"),
+        email: str = Argument(..., help="Email address where to send progress notification"),
         empty: bool = Option(True, help="If true check out empty bucket"),
         debug: bool = Option(False, help="Print complete response to console"),
-        endpoint: str = Option('', help="The endpoint to use for the API")
+        endpoint: str = Option('', help="The endpoint to use for the API. Default is bevaring.digitalarkivet.no"),
```

Comment:
Kunne vi hentet default verdien fra faktisk variabel?
```suggestion
        endpoint: str = Option('', help=f"The endpoint to use for the API. Default is {INSERT_VARIABEL_HERE}"),
```

Reply:
Fungere å kalle på endpoint inne i funksjonen, men får ikke helt til å hente fra faktisk variabel.  "Default is <enterprython._inject._Setting object at 0x1069c6d40> " e det nærmeste æ kommer med min forståelse av dependency injection. Kanskje @PawelMatrasArkivverket har et forslag?

Reply:
Fant en løsning sammen med @PawelMatrasArkivverket